R/utils-omnipath.R

```diff
@@ -155,20 +165,39 @@ get_collectri <- function(organism='human', split_complexes=FALSE, ...){
           stringr::str_detect(source_genesymbol, "NFKB") ~ "NFKB")
       )
   }
-
-  collectri <- base::rbind(collectri_interactions, collectri_complex) %>%
-    dplyr::distinct(source_genesymbol, target_genesymbol,
-                    .keep_all = TRUE) %>%
-    dplyr::mutate(weight = dplyr::case_when(
-      is_stimulation == 1 ~ 1,
-      is_stimulation == 0 ~ -1
-    )) %>%
-    dplyr::select(source_genesymbol, target_genesymbol,
-                  weight) %>%
-    dplyr::rename("source" = source_genesymbol,
-                  "target" = target_genesymbol,
-                  "mor" = weight,
-                  )
+                  
+  if (!load_meta){
```

Comment:
This is a bit too much duplication, only little differences between the two branches

Reply:
Perfect, I think we can merge this :)